tests/core_tests/model_tests/position_dynamics_model_test.py

```diff
@@ -0,0 +1,59 @@
+import unittest
+from core.models.model_list import PositionDynamics
+from core.state import State
+import math
+
+s_2 = {
+    "ang_vel_x": 0.0,
+    "ang_vel_y": 0.0,
+    "ang_vel_z": 0.0,
+    "gnc_pos_q1": 0.0,
+    "gnc_pos_q2": 0.0,
+    "gnc_pos_q3": 0.0,
+    "gnc_pos_q4": 0.0,
+    "vel_x": 1.0,
+    "vel_y": 2.0,
+    "vel_z": 3.0,
+    "x": 10.0,
+    "y": 20.0,
+    "z": 30.0,
+    "force_propulsion_thrusters": 0.0,
+    "fuel_mass": 0.0,
+    "force_earth": 0.0,
+    "force_moon": 0.0,
+    "propulsion_on": False,
+    "solenoid_actuation_on": False,
+}
```

Comment:
Can we import these state dictionaries from a another test instead of re-defining it for this test?

Reply:
If that's the case, it might make sense to move state dictionaries into a separate file called `test_utils.py` or something like that